### ISSUE TITLE
Hide password in bottom bar

### ISSUE THREAD

**Issue by AlexGCWong** [komsit37/sublime-q#40]
When you create a connection and then connect to it, the password is visible in plain text at the bottom. Can this either be omitted or replaced with asterisks?

**Comment by komsit37** [comment]
Yes, this should be easy to do. I won't have time to do this for now. But if anyone wants to tackle this. The code to change should be in this file https://github.com/komsit37/sublime-q/blob/master/q_menu.py. Look for the line with set_status.

**Comment by komsit37** [comment]
fix in current release version
https://github.com/komsit37/sublime-q/pull/44
